src/main/cpp/SwerveDrive.cpp

```diff
@@ -1,122 +1,122 @@
 #include "SwerveDrive.h"
-#include <iostream>
-//Constructor
-SwerveDrive::SwerveDrive(){}
 
+#include <iostream>
+// Constructor
+SwerveDrive::SwerveDrive() {}
 
-//Drive function contains Swerve Inverse Kinematics
-void 
-SwerveDrive::Drive(double x1, double y1, double x2, double rot, bool fieldOrient){
-    double r = sqrt(m_L*m_L + m_W*m_W);
+// Drive function contains Swerve Inverse Kinematics
+void SwerveDrive::Drive(double x1, double y1, double x2, double rot,
+                        bool fieldOrient) {
+    double r = sqrt(m_L * m_L + m_W * m_W);
     y1 *= -1;
     x1 *= -1;
     x2 *= -1;
-    rot *= M_PI/180;
+    rot *= M_PI / 180;
 
-    if(fieldOrient){
-        m_temp = y1*cos(rot) + x1*sin(rot);
-        x1 = -y1*sin(rot) + x1*cos(rot);
+    if (fieldOrient) {
+        m_temp = y1 * cos(rot) + x1 * sin(rot);
+        x1 = -y1 * sin(rot) + x1 * cos(rot);
         y1 = m_temp;
     }
 
-    double a = x1 - x2 * (m_L/r);
-    double b = x1 + x2 * (m_L/r);
-    double c = y1 - x2 * (m_W/r);
-    double d = y1 + x2 * (m_W/r);
+    double a = x1 - x2 * (m_L / r);
+    double b = x1 + x2 * (m_L / r);
+    double c = y1 - x2 * (m_W / r);
+    double d = y1 + x2 * (m_W / r);
 
-    double backRightSpeed = sqrt((a*a) + (d*d));
-    double backLeftSpeed = sqrt((a*a) + (c*c));
-    double frontRightSpeed = sqrt((b*b) + (d*d));
-    double frontLeftSpeed = sqrt((b*b) + (c*c));
+    double backRightSpeed = sqrt((a * a) + (d * d));
+    double backLeftSpeed = sqrt((a * a) + (c * c));
+    double frontRightSpeed = sqrt((b * b) + (d * d));
+    double frontLeftSpeed = sqrt((b * b) + (c * c));
 
-    double backRightAngle = atan2(a, d) * 180/ M_PI;
-    double backLeftAngle = atan2(a, c) *180/ M_PI;
-    double frontRightAngle = atan2(b, d) *180/ M_PI;
-    double frontLeftAngle = atan2(b, c) *180/ M_PI;
+    double backRightAngle = atan2(a, d) * 180 / M_PI;
+    double backLeftAngle = atan2(a, c) * 180 / M_PI;
+    double frontRightAngle = atan2(b, d) * 180 / M_PI;
+    double frontLeftAngle = atan2(b, c) * 180 / M_PI;
 
-    if(x2 == 0 && x1 == 0 && y1 == 0){
+    if (x2 == 0 && x1 == 0 && y1 == 0) {
         m_backRight.Stop();
         m_backLeft.Stop();
         m_frontRight.Stop();
         m_frontLeft.Stop();
         return;
     }
 
-    m_backRight.drive(-backRightSpeed, backRightAngle,
-        DriveConstants::BROFF);
-    m_backLeft.drive(backLeftSpeed, backLeftAngle,
-        DriveConstants::BLOFF);
+    m_backRight.drive(-backRightSpeed, backRightAngle, DriveConstants::BROFF);

```

Comment:
These speed shouldn't need to be inverted, this indicates your modules are 180deg out of tune.

Reply:
Per my definition of tuned: 0rad, positive power, drives the robot forward (+x direction in robot frame)